packages/fluentui/react-northstar/src/themes/teams/components/Attachment/attachmentStyles.ts

```diff
@@ -33,6 +33,21 @@ const attachmentStyles: ComponentSlotStylesPrepared<AttachmentStylesProps, Attac
 
       ...borderFocusStyles,
 
+      '& .ui-attachment__content': {
+        flex: 1,
+      },
+
+      '& .ui-attachment__progress': {
```

Comment:
It's not obvious why this is implemented with selector. I would vote for a separate component, to keep it consistent with other slots..